decidim-proposals/app/views/decidim/proposals/admin/proposals/_js-callout.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="callout callout--full <%= css %>" data-closable>
+  <%= text %>
+  <button class="close-button" aria-label="Dismiss alert" type="button" data-close="">
```

Comment:
Remove the `aria-label` part, or move it to i18n

Reply:
removed